packages/docs/src/components/Layout.tsx

```diff
@@ -67,9 +61,11 @@ const Layout = ({
       </Helmet>
 
       <SkipNav href="#main" />
-      <Header />
+
+      <UsaBanner className="ds-u-display--none ds-u-md-display--block" />
+
       <div className="ds-l-row ds-u-margin--0">
-        <Sidebar isMobileNavOpen={isMobileNavOpen} location={location} />
+        <Navigation location={location} />
```

Comment:
I like that this "Navigation" component now handles all of it. I think that's a good boundary for these components and responsibilities.